package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "$schema": "https://raw.githubusercontent.com/sourcegraph/sourcegraph/master/shared/src/schema/extension.schema.json",
+  "name": "pull-request",
+  "description": "Issues pull request to your code host (GitHub supported currently)",
+  "publisher": "sourcegraph",
+  "activationEvents": [
+    "*"
+  ],
+  "wip": true,
+  "categories": [],
+  "tags": [],
+  "contributes": {
+    "actions": [
+      {
+        "id": "pr.issuePullRequest",
+        "command": "pr.issuePullRequest",
+        "actionItem": {
+          "label": "Pull Request",
+          "description": "Issue a Pull Request"
+        }
+      }
+    ],
+    "menus": {
+      "editor/title": [
+        {
+          "action": "pr.issuePullRequest",
+          "when": "resource"
+        }
+      ],
+      "commandPalette": []
+    },
+    "configuration": {}
+  },
+  "version": "0.0.0-DEVELOPMENT",
+  "license": "MIT",
+  "main": "dist/pull-request.js",
+  "scripts": {
+    "tslint": "tslint -p tsconfig.json './src/**/*.ts'",
```

Comment:
Add prettier and https://npmjs.com/@sourcegraph/prettierrc for code formatting

Reply:
Cool! Didn't know about this.